packages/wallet/app/components/common/Touchable.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import {
+  Platform,
+  TouchableNativeFeedback,
+  TouchableWithoutFeedback,
+  View,
+  TouchableHighlight,
+} from "react-native";
+import { ANDROID_VERSION_LOLLIPOP } from "../../config/constants";
+
+type TExternalProps = React.ComponentProps<typeof TouchableWithoutFeedback> & {
+  activeColor: string;
+  children: React.ReactNode;
+};
+
+const isRippleSupported = Platform.OS === "android" && Platform.Version >= ANDROID_VERSION_LOLLIPOP;
+
+/**
+ * A wrapper for components that should respond to touches.
+ * Provides a material "ink ripple" interaction effect for supported platforms (>= Android Lollipop).
+ * On unsupported platforms, it falls back to a highlight effect.
+ */
+const Touchable = React.forwardRef<TouchableHighlight, TExternalProps>(
+  ({ children, style, activeColor, ...props }, ref) => {
+    if (isRippleSupported) {
+      // TODO: There is a bug in react-native were ripple effect do not follow the borderRadius
+      // see https://github.com/facebook/react-native/issues/6480
+      return (
+        <TouchableNativeFeedback
+          ref={ref}
+          background={TouchableNativeFeedback.Ripple(activeColor)}
+          {...props}
+        >
+          <View style={style}> {React.Children.only(children)}</View>
```

Comment:
not sure if this extra space will show up in render

```suggestion
          <View style={style}>{React.Children.only(children)}</View>
```